lib/_stream_writable.js

```diff
@@ -108,6 +108,33 @@ function WritableState(options, stream) {
 
   // True if the error was already emitted and should not be thrown again
   this.errorEmitted = false;
+
+  // count buffered requests
+  this.bufferedRequestCount = 0;
+
+  this.corkedRequestsFree = new CorkedRequest(this);
+  this.corkedRequestsFree.next = new CorkedRequest(this);
+}
+
+function CorkedRequest(state) {
```

Comment:
It's not a linked list. It's two elements linked together. They are rotated and nothing is added. Each corked request contains a linked list of entries. I though naming it as a list was bad, but I'm open to suggest to clear readability.